api/v1beta2/ibmpowervscluster_webhook.go

```diff
@@ -96,60 +90,3 @@ func (r *IBMPowerVSCluster) validateIBMPowerVSClusterNetwork() *field.Error {
 	}
 	return nil
 }
-
-func (r *IBMPowerVSCluster) validateIBMPowerVSClusterCreateInfraPrereq() *field.Error {
-	annotations := r.GetAnnotations()
-	if len(annotations) == 0 {
-		return nil
-	}
-
-	value, found := annotations[CreateInfrastructureAnnotation]
-	if !found {
-		return nil
-	}
-
-	createInfra, err := strconv.ParseBool(value)
-	if err != nil {
-		return field.Invalid(field.NewPath("annotations"), r.Annotations, "value of powervs.cluster.x-k8s.io/create-infra should be boolean")
-	}
-
-	if !createInfra {
-		return nil
-	}
-
-	if r.Spec.Zone == nil {
-		return field.Invalid(field.NewPath("spec.zone"), r.Spec.Zone, "value of zone is empty")
-	}
-
-	if r.Spec.VPC == nil {
-		return field.Invalid(field.NewPath("spec.vpc"), r.Spec.VPC, "value of VPC is empty")
-	}
-
-	if r.Spec.VPC.Region == nil {
-		return field.Invalid(field.NewPath("spec.vpc.region"), r.Spec.VPC.Region, "value of VPC region is empty")
-	}
-
-	if r.Spec.ResourceGroup == nil {
-		return field.Invalid(field.NewPath("spec.resourceGroup"), r.Spec.ResourceGroup, "value of resource group is empty")
-	}
-
-	if r.Spec.Ignition == nil {
-		return nil
-	}
```

Comment:
Hey, You are supposed remove it from here, Not the whole file, We still need webhook for verifying zone vpc region.